### PR TITLE
lab11-Python, lab01-Django, lab02 part B, lab03 part A and B-JS, lab04 and lab03-html done

### PR DESCRIPTION
**Pull request by juliachobu**
Still working on different labs

### CODE REVIEW

File: code/julia/Javascript/lab01_A/part_A.js [hunk 1]
```diff
@@ -0,0 +1,20 @@
+// Unit converter
```
Comment (brucestull) [inline code/julia/Javascript/lab01_A/part_A.js:1]: **Needs Work**
* Entered `6` and `ft` and `inches`:
    * Expected about 72 inches.
    * Result is `1.82 inches`.
image

File: code/julia/Javascript/lab02_A/part_A.js [hunk 2]
```diff
@@ -0,0 +1,28 @@
+//Blackjack Advice
```
Comment (brucestull) [inline code/julia/Javascript/lab02_A/part_A.js:1]: **Needs Work**
* Entered:
    * `ace`
    * `10`
    * `2`
    * Expected something like 'bust':
    * Received no output.

**Comment by brucestull** [comment]
[Lab 11: Quotes API](https://github.com/PdxCodeGuild/class_swordfish/blob/main/1%20Python/labs/11%20Quotes%20API.md)
**Needs Work**
* 'new search' gives unexpected results.
* Sample output:
```
Enter 'next page' for more quotes, 'new search' to enter a new search keyword or 'exit' to exit:
 >>> new search
Enter a new keyword:
cat
Marc Jacobs 
 I remember walking the dog one day, I saw a car full of teenage girls, and one of them rolled down the window and yelled, 'Marc Jacobs!' in a French accent. 

Marc Jacobs 
 I remember walking the dog one day, I saw a car full of teenage girls, and one of them rolled down the window and yelled, 'Marc Jacobs!' in a French accent. 

Marc Jacobs 
 I remember walking the dog one day, I saw a car full of teenage girls, and one of them rolled down the window and yelled, 'Marc Jacobs!' in a French accent. 

Marc Jacobs 
 I remember walking the dog one day, I saw a car full of teenage girls, and one of them rolled down the window and yelled, 'Marc Jacobs!' in a French accent.

Marc Jacobs
 I remember walking the dog one day, I saw a car full of teenage girls, and one of them rolled down the window and yelled, 'Marc Jacobs!' in a French accent.

Marc Jacobs
 I remember walking the dog one day, I saw a car full of teenage girls, and one of them rolled down the window and yelled, 'Marc Jacobs!' in a French accent.

Marc Jacobs
 I remember walking the dog one day, I saw a car full of teenage girls, and one of them rolled down the window and yelled, 'Marc Jacobs!' in a French accent.

Marc Jacobs
 I remember walking the dog one day, I saw a car full of teenage girls, and one of them rolled down the window and yelled, 'Marc Jacobs!' in a French accent.

Marc Jacobs
 I remember walking the dog one day, I saw a car full of teenage girls, and one of them rolled down the window and yelled, 'Marc Jacobs!' in a French accent.

Marc Jacobs
 I remember walking the dog one day, I saw a car full of teenage girls, and one of them rolled down the window and yelled, 'Marc Jacobs!' in a French accent.

Marc Jacobs
 I remember walking the dog one day, I saw a car full of teenage girls, and one of them rolled down the window and yelled, 'Marc Jacobs!' in a French accent.

Marc Jacobs
 I remember walking the dog one day, I saw a car full of teenage girls, and one of them rolled down the window and yelled, 'Marc Jacobs!' in a French accent.

Marc Jacobs
 I remember walking the dog one day, I saw a car full of teenage girls, and one of them rolled down the window and yelled, 'Marc Jacobs!' in a French accent.

Marc Jacobs
 I remember walking the dog one day, I saw a car full of teenage girls, and one of them rolled down the window and yelled, 'Marc Jacobs!' in a French accent.

Marc Jacobs
 I remember walking the dog one day, I saw a car full of teenage girls, and one of them rolled down the window and yelled, 'Marc Jacobs!' in a French accent.

Marc Jacobs
 I remember walking the dog one day, I saw a car full of teenage girls, and one of them rolled down the window and yelled, 'Marc Jacobs!' in a French accent.

Marc Jacobs
 I remember walking the dog one day, I saw a car full of teenage girls, and one of them rolled down the window and yelled, 'Marc Jacobs!' in a French accent.

Marc Jacobs
 I remember walking the dog one day, I saw a car full of teenage girls, and one of them rolled down the window and yelled, 'Marc Jacobs!' in a French accent.

Marc Jacobs
 I remember walking the dog one day, I saw a car full of teenage girls, and one of them rolled down the window and yelled, 'Marc Jacobs!' in a French accent.

Marc Jacobs
 I remember walking the dog one day, I saw a car full of teenage girls, and one of them rolled down the window and yelled, 'Marc Jacobs!' in a French accent.

Marc Jacobs
 I remember walking the dog one day, I saw a car full of teenage girls, and one of them rolled down the window and yelled, 'Marc Jacobs!' in a French accent.

Marc Jacobs
 I remember walking the dog one day, I saw a car full of teenage girls, and one of them rolled down the window and yelled, 'Marc Jacobs!' in a French accent.

Marc Jacobs
 I remember walking the dog one day, I saw a car full of teenage girls, and one of them rolled down the window and yelled, 'Marc Jacobs!' in a French accent.

Marc Jacobs
 I remember walking the dog one day, I saw a car full of teenage girls, and one of them rolled down the window and yelled, 'Marc Jacobs!' in a French accent.

Marc Jacobs
 I remember walking the dog one day, I saw a car full of teenage girls, and one of them rolled down the window and yelled, 'Marc Jacobs!' in a French accent.

Enter 'next page' for more quotes, 'new search' to enter a new search keyword or 'exit' to exit:
 >>>
```

**Review by brucestull — CHANGES_REQUESTED** [review]
[Lab 8: Contact List](https://github.com/PdxCodeGuild/class_swordfish/blob/main/1%20Python/labs/08%20Contact%20List.md)
**Needs Work**
* See inline code comments.

[Lab 9: ATM](https://github.com/PdxCodeGuild/class_swordfish/blob/main/1%20Python/labs/09%20ATM.md)
**Needs Work**
* See inline code comments.

[Lab 10: Dad Joke API](https://github.com/PdxCodeGuild/class_swordfish/blob/main/1%20Python/labs/10%20Dad%20Joke%20API.md)
**Completed**

[Lab 11: Quotes API](https://github.com/PdxCodeGuild/class_swordfish/blob/main/1%20Python/labs/11%20Quotes%20API.md)
**Needs Work**
* See comments section:
    * 'new search' function doesn't work as expected.

[Lab 1: Bio](https://github.com/PdxCodeGuild/class_swordfish/blob/main/2%20Flask%20%2B%20HTML%20%2B%20CSS/labs/01%20Bio.md)
**Completed**

[Lab 1: Grocery List](https://github.com/PdxCodeGuild/class_swordfish/blob/main/3%20Django/labs/01%20Grocery%20List.md)
**Needs Work**
* See inline code comments.

[Lab 1-3: Pick 3](https://github.com/PdxCodeGuild/class_swordfish/blob/main/4%20JavaScript/labs/Lab%2001-03%20Pick%203.md)
**Lab 01 A** - Unit Converter:
* **Needs Work**
* See inline comments in code.

**Lab 02 A** - Blackjack Advice:
* **Needs Work**
* See inline comments in code.

**Lab 03 A** - BlackJack Advice:
* **Needs Work**
* Lab needs to be some python lab other than the two chosen above (Unit Converter, BlackJack Advice).

**Lab 01 B** - Unit Converter:
* **Needs Work**
* In progress.

**Lab 02 B**:
* **Missing**

**Lab 03 B**:
* **Missing**

**Comment by brucestull** [comment]
[Lab 8: Contact List](https://github.com/PdxCodeGuild/class_swordfish/blob/main/1%20Python/labs/08%20Contact%20List.md)
**Completed**
* Student demonstrated functionality.

**Comment by brucestull** [comment]
[Lab 1-3: Pick 3](https://github.com/PdxCodeGuild/class_swordfish/blob/main/4%20JavaScript/labs/Lab%2001-03%20Pick%203.md)
Part A:
* Lab01_Unitconverter:
    * **Completed**
*

**Review by Faebaeee — APPROVED** [review]
Great job!
[Lab 2: Company Home][(https://github.com/PdxCodeGuild/class_swordfish/blob/main/2%20Flask%20%2B%20HTML%20%2B%20CSS/labs/02%20Company.md)
Lab 2 Html/Css Complete

**Review by Faebaeee — APPROVED** [review]
Merging to clean up pull request

**Pull request merged**